local.tf

```diff
@@ -57,6 +57,15 @@ locals {
             "${local.alertmanager.domain}",
             "alertmanager.apps.${var.base_domain}"
           ]
+          tls = [
+            {
+              secretName = "alertmanager-tls"
+              hosts = [
+                "${local.alertmanager.domain}",
```

Comment:
In terraform 0.x, we used to get a warning `Interpolation-only expressions are deprecated` with this kind of syntax. I see that it's not the case anymore with terraform >= 1.0. Does it mean that it's good practice now?

Reply:
I don’t think so, I guess it’s a copy/paste of the lines above. If we want to change it, I suggest doing it in the entire file in a separate format PR.